src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/documents/dossier/DossierContextFactoryTest.kt

```diff
@@ -98,6 +100,96 @@ class DossierContextFactoryTest {
 
     val result = underTest.createContext(recallId)
 
-    assertThat(result, equalTo(DossierContext(recall, currentPrisonName, currentPrisonIsWelsh, 1, STANDARD)))
+    assertThat(result, equalTo(DossierContext(recall, currentPrisonName, currentPrisonIsWelsh, 1)))
+  }
+
+  @Test
+  fun `get TableOfContentsContext for standard recall doesnt calculate recall length`() {
```

Comment:
How about covering the tests in the class without so much test code duplication?  There looks to be a lot in this as it is.

Reply:
Could do 🤷‍♂️

Reply:
We can leave but the number, combinations and duplication between some of the doc testing is starting to bovver me a bit.